applications/matter_bridge/src/bridge_shell.cpp

```diff
@@ -47,8 +47,10 @@ static void AddDevice(const struct shell *shell, int deviceType, const char *nod
 	VerifyOrReturn(provider != nullptr,
 		       shell_fprintf(shell, SHELL_INFO, "Cannot allocate data provider of given type\n"));
 
-	auto *newBridgedDevice = BridgeFactory::GetBridgedDeviceFactory().Create(
-		static_cast<BridgedDevice::DeviceType>(deviceType), nodeLabel);
+	auto *newBridgedDevice =
+		BridgeFactory::GetBridgedDeviceFactory().Create(static_cast<BridgedDevice::DeviceType>(deviceType),
+								nodeLabel,
```

Comment:
Hmm, does the create function require providing the `deviceType` argument twice?

Reply:
Yeah, it looks weird :) It is because the first argument of `Create` method is used to determine the type in creation map, and all of the following arguments are forwarded to the specific device constructor. 
I'm not sure yet how to fix this, let me think about it.

Reply:
Fixed, I modified the constructor to not contain the `deviceType` argument. The device type will be held in the bridged device, as a const value of `BridgedDevice::DeviceType` enum.